libdnf/module/ModulePackageContainer.hpp

```diff
@@ -142,7 +142,7 @@ struct ModulePackageContainer
     *
     * @return bool
     */
-    bool enable(const std::string &name, const std::string &stream);
+    bool enable(const std::string &name, const std::string &stream, const bool count = true);
```

Comment:
@kontura Please could you add a description to the `count` argument not only in this method?

Reply:
Added a description, please check if its understandable.

Reply:
What about ?
 ```
When the count parameter is set to false the change will be not count towards the limit of module state modification
```
But I am not sure.

Reply:
The documentation should be also enhanced in disable and reset methods.

Reply:
ok, updated the documentation for enable, disable and reset with basically the version you suggested.